handler_models.go

```diff
@@ -257,14 +256,13 @@ func ModelOwnerVersionZip(owner, name string, user *users.User, tx *gorm.DB,
 	// commit the DB transaction
 	// Note: we commit the TX here on purpose, to be able to detect DB errors
 	// before writing "data" to ResponseWriter. Once you write data (not headers)
-	// into it the status code is set to 200 (OK).
 	if err := tx.Commit().Error; err != nil {
 		return nil, gz.NewErrorMessageWithBase(gz.ErrorZipNotAvailable, err)
 	}
 
 	// Redirect to the cloud storage
-	http.Redirect(w, r, *zipPath, http.StatusOK)
-	return nil, nil
+	http.Redirect(w, r, *zipPath, http.StatusSeeOther)
```

Comment:
Consider using `302 http.StatusFound` instead of `303 http.StatusSeeOther`.

While they are functionally identical for our use case, the 303 code implies that the request is being redirected to another page to confirm/handle the request and access the resource, whereas the 302 request tells the client that the resource is in the returned address.